getting-started/templates/systemlink-values.yaml

```diff
@@ -537,10 +537,50 @@ dataframeservice:
             <value>false</value>
           </property>
 
+  ## Configure Kafka cluster
```

Comment:
Do we expect the user to do something with this commented out configuration? If so, we should add guidance as comments.

Reply:
Speaking with @spanglerco - We actually decided to leave out those configurations for now until we have a more consistent message for metrics collection across the platform.